### PR TITLE
fixed styles in not found page

### PR DESCRIPTION
**Pull request by asseemm**
image

### CODE REVIEW

File: client/src/pages/404.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+const NotFoundPage = () => {
+  return (
+    <NotFoundWrapper>
+      <Flexbox direction="column" align="center">
+        <Title>404</Title>
+        <Subtitle>Not found</Subtitle>
+        <Instruction>
+          Эта страница недоступна. <br /> Вернитесь на{' '}
+          <a href="/">главную страницу</a> нашего сайта.
```
Comment (eyesofdaveed) [inline client/src/pages/404.jsx]: use `LinkRouter` from react-router-dom, to actually navigate an user to the home page.

File: client/src/atoms/ProfiPerformance.jsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import Demo from './demo';
+
+createRoot(document.getElementById('container')).render(<Demo />);
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiPerformance.jsx]: This doesn't look right, you don't need to create an another route just for a single atom.

Comment (asseemm) [reply client/src/atoms/ProfiPerformance.jsx]: Это не мой файл, случайно добавился после pull, удалила.

File: client/src/pages/404.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Link } from 'react-router-dom';
+import { Flexbox } from '../atoms/Flexbox';
+import { colors } from '@base';
```
Comment (eyesofdaveed) [inline client/src/pages/404.jsx]: `
import React from 'react';
import styled from 'styled-components';
import { Link } from 'react-router-dom';

import { Flexbox } from '@atoms';
import { colors } from '@base';
`

File: client/src/pages/404.jsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+const NotFoundWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+`;
+
+const Title = styled.p`
```
Comment (eyesofdaveed) [inline client/src/pages/404.jsx]: create `Title` atom, that will have by default this styles

File: client/src/pages/404.jsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  color: ${colors.cardBg};
+  font-size: 150px;
+  font-weight: bold;
+
+  @media screen and (max-width: 450px) {
+    font-size: 90px;
+  }
+`
+const Subtitle = styled.p`
```
Comment (eyesofdaveed) [inline client/src/pages/404.jsx]: const SubTitle = styled(Title)`
  font-size: 30px;
  font-weight: bold;
`

File: client/src/pages/404.jsx (hunk 4 above)
Comment (eyesofdaveed) [reply client/src/pages/404.jsx]: `const StyledTitle = styled(Title)`
`const SubText = styled.p`

**Review by eyesofdaveed — APPROVED** [review]

**Pull request merged**